src/lib/gssapi/krb5/accept_sec_context.c

```diff
@@ -424,6 +424,25 @@ kg_process_extension(krb5_context context,
 /* A zero-value channel binding, for comparison */
 static const uint8_t null_cb[CB_MD5_LEN];
 
+static krb5_error_code
+check_ap_options_cbt(krb5_context context, krb5_authdata **authdata,
+                     krb5_boolean *client_asserted_cd)
```

Comment:
shouldn't this be client_asserted_cb ?
(for channel bindings I assume ?)

Reply:
Yeah typo, nice catch :)